libioc/Network.py

```diff
@@ -211,68 +220,50 @@ def epair_id(self) -> int:
 
     def __create_new_epair_interface(
         self,
-        variable_name_a: str,
-        variable_name_b: str,
         nic_suffix_a: str=":a",
         nic_suffix_b: str=":b",
         **nic_args: typing.Any
-    ) -> CreatedCommandList:
-
-        commands: CreatedCommandList = []
+    ) -> typing.Tuple[
+        libioc.NetworkInterface.NetworkInterface,
+        libioc.NetworkInterface.NetworkInterface
+    ]:
 
-        commands += libioc.NetworkInterface.QueuingNetworkInterface(
+        nic_a = libioc.NetworkInterface.NetworkInterface(
             name="epair",
             create=True,
-            shell_variable_nic_name=variable_name_a,
             logger=self.logger
-        ).read_commands()
-
-        commands.append(
-            f"export {variable_name_b}="
-            f"$(echo ${variable_name_a} | sed 's/.$/b/')"
         )
+        nic_b_name = nic_a.name[:-1] + "b"
 
-        epair_a = libioc.NetworkInterface.QueuingNetworkInterface(
-            name=None,
+        libioc.NetworkInterface.NetworkInterface(
+            name=nic_a,
```

Comment:
can you briefly elaborate what it is that makes the queueing of commands unnecessary now?

Reply:
Queuing* was a hack around our inability to execute right away. The Queuing wrapper class collected all calls to self._exec and presented the list of commands at a later time. From those lists, the jail hooks have been previously generated. They have been made from minute 1 to get deprecated in a moment like this.